packages/opentelemetry-plugin-xml-http-request/test/unmocked.test.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { XMLHttpRequestPlugin } from '../src';
+import { ReadableSpan, SpanProcessor } from '@opentelemetry/tracing';
+import { WebTracerProvider } from '@opentelemetry/web';
+import assert = require('assert');
+import { HttpAttribute } from '@opentelemetry/semantic-conventions';
+
+class SpanCapturer implements SpanProcessor {
```

Comment:
can you please use the name "Processor" instead of Capturer ?, maybe `TestSpanProcessor` ?

Reply:
Sure thing; I like `TestSpanProcessor` too.